fcrepo-kernel-modeshape/src/test/java/org/fcrepo/integration/kernel/modeshape/FedoraResourceImplIT.java

```diff
@@ -1098,6 +1100,43 @@ public void testDeletePartOfMultiValueProperty() throws RepositoryException {
                 createResource("info:fedora/" + pid + "/c")));
     }
 
+    @Test
+    public void testGetTimeMapLDPCv() throws RepositoryException {
+        final String pid = getRandomPid();
+        final Session jcrSession = getJcrSession(session);
+        final FedoraResource resource = containerService.findOrCreate(session, "/" + pid);
+        session.commit();
+
+        // Create TimeMap (LDPCv)
+        final FedoraResource ldpcvResource = resource.findOrCreateLDPCv();
+
+        assertNotNull(ldpcvResource);
+        assertEquals("/" + pid + "/" + LDPCV_TIME_MAP, ldpcvResource.getPath());
+
+        session.commit();
+
+        assertEquals(jcrSession.getNode("/" + pid).getNode(LDPCV_TIME_MAP),
+                ((FedoraResourceImpl)resource.getLDPCv()).getNode());
+    }
+
+    @Test
+    public void testFindOrCreateTimeMapLDPCv() throws RepositoryException {
+        final String pid = getRandomPid();
+        final Session jcrSession = getJcrSession(session);
+        final FedoraResource resource = containerService.findOrCreate(session, "/" + pid);
+        session.commit();
+
+        // Create TimeMap (LDPCv)
+        final FedoraResource ldpcvResource = resource.findOrCreateLDPCv();
+
+        assertNotNull(ldpcvResource);
+        assertEquals("/" + pid + "/" + LDPCV_TIME_MAP, ldpcvResource.getPath());
+
+        session.commit();
+
+        assertTrue(jcrSession.getNode("/" + pid).getNode(LDPCV_TIME_MAP).isNodeType(FEDORA_TIME_MAP));
```

Comment:
You can probably just add this line to the test above and then remove this otherwise duplicative test method.